docs/README.md

```diff
@@ -195,18 +195,20 @@ typescript_bin.tsc(
 )
 ```
 
-> Note: this doesn't cause an eager fetch!
-> Bazel doesn't download the typescript package when loading this file, so you can safely write this
-> even in a BUILD.bazel file that includes unrelated rules.
+Each bin exposes three rules, one for each Bazel command ("verb"): build, test and run - each aligning with the corresponding [js_run_binary](./js_run_binary.md), [js_test](#js_test) and [js_binary](./js_binary.md) rule APIs.
 
-Each bin exposes three rules, one for each Bazel command ("verb"):
+For example:
 
 | Use          | With          | To              |
```

Comment:
Looking better. Thanks for updating. Can you add a column to the table as to which underlying rule is used?